PerDiem/PerDiem/Controllers/CalendarTransition.m

```diff
@@ -35,17 +36,33 @@ - (void)animateTransition:(id <UIViewControllerContextTransitioning>)transitionC
     
     CalendarDayViewController *dayVc;
     CalendarViewController *monthVc;
-    
+
+    // This doesn't too nice, and also the transition I made is not consistent with the rest of the system.
+    if ([[transitionContext viewControllerForKey:UITransitionContextToViewControllerKey] isKindOfClass:[TransactionFormViewController class]] ||
+        [[transitionContext viewControllerForKey:UITransitionContextFromViewControllerKey] isKindOfClass:[TransactionFormViewController class]]) {
+        UIViewController *toViewController = [transitionContext viewControllerForKey:UITransitionContextToViewControllerKey];
+        [[transitionContext containerView] addSubview:toViewController.view];
+        toViewController.view.alpha = 0.0;
+        [UIView animateWithDuration:0.8 animations:^{
+            toViewController.view.alpha = 1.0;
+            if ([toViewController isKindOfClass:[TransactionFormViewController class]]) {
+                toViewController.navigationController.navigationBar.alpha = 1;
```

Comment:
I think you need to set `navigationBar.translucent = NO;` to make the form not appear below the navbar. like this: https://github.com/PerDiem/per-diem/blob/master/PerDiem/PerDiem/Controllers/NavigationViewController.m#L25